cdap-docs/user-guide/source/pipelines/plugins/pom.xml

```diff
@@ -20,7 +20,7 @@
 
   <groupId>io.cdap.hydrator</groupId>
   <artifactId>hydrator-plugins</artifactId>
-  <version>1.7.1-SNAPSHOT</version>
```

Comment:
Is this expected?

Reply:
Thanks for catching that! Its not intended, fixed